### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by Ippey**
## Type of Change

- **Something else:** Add Japanese translation.

## What issue does this relate to?

None

### What should this PR do?

- Add Japanese translation.
- Add `ja` in each `languages.js` .

### CODE REVIEW

**Comment by MattIPv4** [comment]
:wave: Hey @Ippey, would you mind updating the `Copyright xxxx` value in the files you've modified to be 2022?

I've also just merged #366 which added two new strings that your language pack will need 💙 

Tyvm!

**Comment by Ippey** [comment]
@MattIPv4 I've just modified it!

**Comment by Ippey** [comment]
I'm sorry. I added `disableHtmlCaching`, `enableDisableHtmlCaching`  translation in `ja`. 🙏

**Comment by SiderealArt** [comment]
Hi, could you take a look at the suggestions I made above?

**Comment by SiderealArt** [comment]
Yes, as @MattIPv4 said earlier.

**Comment by Ippey** [comment]
@MattIPv4  I also modified every `languages.js`.

Thank you, @SiderealArt !

**Comment by SiderealArt** [comment]
I forgot to click comment. I meant these suggestions. Sorry for bothering you again. 😥

File: src/nginxconfig/i18n/ja/templates/setup_sections/go_live.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+import common from '../../common';
+
+export default {
+    letsGoLive: 'Let\'s go live!',
```
Comment (MattIPv4) [inline src/nginxconfig/i18n/ja/templates/setup_sections/go_live.js:30]: Can this be translated?

File: src/nginxconfig/i18n/ja/templates/setup_sections/go_live.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    reloadNginxToLoadInYourNewConfiguration: `${common.nginx} をリロードして新しい設定を反映します:`,
+    goLive: 'Go live!',
```
Comment (MattIPv4) [inline src/nginxconfig/i18n/ja/templates/setup_sections/go_live.js:32]: Can this be translated?

File: src/nginxconfig/i18n/pl/languages.js [hunk 3]
```diff
@@ -34,4 +34,5 @@ export default {
     ru: 'Rosyjski',
     pl: 'Polski',
     de: 'Niemiecki',
+    ja: 'Japonês',
```
Comment (MattIPv4) [inline src/nginxconfig/i18n/pl/languages.js:37]:
```suggestion
    ja: 'Japoński',
```

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
💙 Thank you for this translation -- a few quick thoughts:

**Comment by Ippey** [comment]
@MattIPv4 @SiderealArt Thank you for your suggestions. I corrected them.

**Review by MattIPv4 — APPROVED** [review]
LGTM, thank you very much for contributing this! 💙

**Comment by MattIPv4** [comment]
Hey, @Ippey - Thanks a ton for this sweet PR! 😄

Would you please shoot me an email when you get a chance?
mcowley at digitalocean dot com 🎉

**Pull request merged**